BUILDING.md

```diff
@@ -9,20 +9,74 @@ If you consistently can reproduce a test failure, search for it in the
 file a new issue.
 
 
+## Supported platforms
+
+This list of supported platforms is current as of the branch / release to which it is attached.
+
+### Input
+
+We rely on a few dependencies that _makes us who we are—most importantly V8 and libuv. We therefore need to adopt their supported platforms and potentially add to their lists based on test and/or release coverage.
+
+### Strategy
+
+Support is divided into three tiers:
+
+* **Tier 1**: Full test coverage and maintenance by the Node.js core team and the broader community.
+* **Tier 2**: Full test coverage but more limited maintenance, often provided by the vendor of the platform. 
+* **Experimental**: Known to compile but not necessarily reliably or with a full passing test suite. These are often working to be promoted to Tier 2 but are not quite ready. There is at least one individual actively providing maintenance and the team is striving to broaden quality and reliability of support.
+
+### Supported platforms
+
+|  System      | Support type | Version                          | Architectures        | Notes            |
+|--------------|--------------|----------------------------------|----------------------|------------------|
+| GNU/Linux    | Tier 1       | kernel >= 2.6.18, glibc >= 2.5   | x86, x64, arm, arm64 |                  | 
+| macOS        | Tier 1       | >= 10.10                         | x64                  |                  |
+| Windows      | Tier 1       | >= Windows 7 or >= Windows2008R2 | x86, x64             |                  |
+| SmartOS      | Tier 2       | = 14                             | x86, x64             |                  |
+| FreeBSD      | Tier 2       | >= 10                            | x64                  |                  |
+| GNU/Linux    | Tier 2       | kernel >= 4.2.0, glibc >= 2.19   | ppc64be              |                  |
+| GNU/Linux    | Tier 2       | kernel >= 3.13.0, glibc >= 2.19  | ppc64le              |                  |
+| AIX          | Tier 2       | >= 6.1                           | ppc64be              |                  |
```

Comment:
Please change to

+| AIX          | Tier 2       | >= 6.1  TL09                     | ppc64be              |                  |